### PR TITLE
KEYCLOAK-16057 KEYCLOAK-16058 Use the latest AMO version

### PR DESCRIPTION
**Pull request by slaskawi**
## JIRA ID

https://issues.redhat.com/browse/KEYCLOAK-16058
https://issues.redhat.com/browse/KEYCLOAK-16057

## Additional Information

This Pull Request revisits the Keycloak Dashboard and brings automatic Application Monitoring Operator installation to our Makefile. 

## Checklist:
- [ ] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

**Review by pb82 — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -7,6 +7,10 @@ OPERATOR_SDK_DOWNLOAD_URL=https://github.com/operator-framework/operator-sdk/rel
 MINIKUBE_DOWNLOAD_URL=https://github.com/kubernetes/minikube/releases/download/v1.9.2/minikube-linux-amd64
 KUBECTL_DOWNLOAD_URL=https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
 
+# Monitoring credentials (for federation)
+_OS_PROMETHEUS_USER := $(shell oc get secrets -n openshift-monitoring grafana-datasources -o 'go-template={{index .data "prometheus.yaml"}}' | base64 --decode | jq -r '.datasources[0].basicAuthUser')
+_OS_PROMETHEUS_PASS := $(shell oc get secrets -n openshift-monitoring grafana-datasources -o 'go-template={{index .data "prometheus.yaml"}}' | base64 --decode | jq -r '.datasources[0].basicAuthPassword')
+
```
Comment (ASzc) [inline Makefile:13]: Having this here would slow down all targets. There may not even normally be an openshift running for some targets.

Comment (slaskawi) [reply Makefile]: Fixed. Thanks for spotting this.

**Comment by slaskawi** [comment]
@ASzc Can I ask you to re-review this one?

**Review by ASzc — APPROVED** [review]
Looks good

**Review by abstractj — APPROVED** [review]

**Pull request merged**